src/theme/variables.scss

```diff
@@ -1,15 +1,16 @@
 @use "sass:color";
 @use "sass:map";
 @use "./breakpoints";
-@use "./themes/default.scss";
+@use "./themes/index" as *;
```

Comment:
nit(non-blocking) - probably safer to keep namespacing, so include statements more like `themes.theme-default`, although I can see we have `theme-` prefixing so not essential